app/app.js

```diff
@@ -95,7 +86,10 @@ app.use(function (req, res, next) {
 // Add variables that are available in all views.
 app.use(function (req, res, next) {
   res.locals.serviceName = serviceName
+  res.locals.organisationName = organisationName
   res.locals.releaseVersion = 'v' + releaseVersion
+  res.locals.serialisedName = nameSerialiser(res.locals.user.name)
```

Comment:
Might be worth a check on `res.locals.user.name` just in case there's no user. Should never happen but you never know 🙂